config/sync/core.entity_view_display.node.gp_practice.default.yml

```diff
@@ -89,6 +84,14 @@ content:
         style: ''
         preferScrollingToZooming: 0
         gestureHandling: auto
+      data_provider_settings: {  }
+      centre:
```

Comment:
What's changed to make this entity spit this extra config out here? It might well be harmless but thought It wise to check

Reply:
I rolled back and made the moderation control change again and these lines were added in the same way. We'll just have to put it down to a config export oddity I think.

Reply:
That's okay. It's on the default view mode so shouldn't affect anything visually. Another reason never to use the default view mode for theming related work!